packages/terra-form-select/src/_Menu.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { polyfill } from 'react-lifecycles-compat';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
Does terra-i18n include this? It feels like this would be better pulled from that than react-intl directly?

Reply:
We want to pull these from react-intl. Pulling in injectIntl and intlShape from terra-i18n has caused some issue in the past, we have this issue logged which covers the issue in more detail: https://github.com/cerner/terra-core/issues/1982

Reply:
sounds good